Mathlib/MeasureTheory/Integral/Average.lean

```diff
@@ -197,17 +197,19 @@ theorem laverage_mem_openSegment_compl_self [IsFiniteMeasure μ] (hs : NullMeasu
 #align measure_theory.laverage_mem_open_segment_compl_self MeasureTheory.laverage_mem_openSegment_compl_self
 
 @[simp]
-theorem laverage_const (μ : Measure α) [IsFiniteMeasure μ] [h : NeZero μ] (c : ℝ≥0∞) :
```

Comment:
Do you really intend this change? Maybe it's a result of some git-foo?

I thought we standardised to `NeZero μ` after https://github.com/leanprover-community/mathlib4/pull/6048

Reply:
Definitely a git mistake on my side. Thanks for noticing!